src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -1943,6 +1943,28 @@ private static int HighNarrowing(long op1, bool round)
 
         public static long MultiplyWideningUpperAndSubtract(long[] op1, int[] op2, int[] op3, int i) => MultiplyWideningAndSubtract(op1[i], op2[i + op2.Length / 2], op3[i + op3.Length / 2]);
 
+        public static T ShiftLeft<T>(T op1, ulong op2) where T : INumber<T>
```

Comment:
can we use the existing `Helpers.ShiftLeftLogical()`?

Reply:
I don't think those match the functionality of these APIs: The existing `UnsignedShift` seems to only consider the first byte of the shift value, regardless of the size of the input, whereas the SVE APIs consider the whole second operand ([docs](https://dougallj.github.io/asil/doc/lsl_z_p_zw_8.html)). I agree the code duplication isn't ideal -- I'm happy to rename the new helpers to something more discerning, if you'd like.